lib/Lift/JumpTargetManager.h

```diff
@@ -596,6 +614,26 @@ class JumpTargetManager {
   MetaAddressSet AVIPCWhiteList;
   const TupleTree<model::Binary> &Model;
   const RawBinaryView &BinaryView;
+
+private:
+  // Helpers to break the harvestWithAVI down.
+  void updateCPUStateAccess();
+  void cloneRoot(llvm::Function **OptimizedFunction,
+                 llvm::ValueToValueMapTy &OldToNew);
+  void registerLoadStoreAddrs(llvm::Function *OptimizedFunction,
+                              AnalysisRegistry &AR);
+  using GlobalToAllocaTy = llvm::DenseMap<llvm::GlobalVariable *,
+                                          llvm::AllocaInst *>;
+  void createAllocaPerCSV(llvm::Function *OptimizedFunction,
+                          GlobalToAllocaTy &CSVMap);
```

Comment:
1. The only caller of `createAllocaPerCSV` instantiates `CSVMap` right before calling this function: make the function return `CSVMap`.
2. Misleading name: it also replaces the uses. Rename to `promoteCSVsToAlloca`.